### ISSUE TITLE
sam.native.yaml template missing

### ISSUE THREAD

**Issue by massimiliano76** [gunnarmorling/search.morling.dev#1]
The sam.native.yaml template is missing in the source code.

**Comment by gunnarmorling** [comment]
Yes, indeed. I got that in separate private repo atm., as I wasn't 100% bought into sharing all its private ids etc. I'll have to think about that.
